File: CSharpSDK/Bean/Common/MrmPair.cs
using System;
using System.Collections.Generic;

namespace AirdSDK.Beans.Common
{
    public class MrmPair
    {

        /**
        * order number for current spectrum
        */
        public int num;

        public string id;

        public string key;

        public WindowRange precursor;
        public WindowRange product;

        public string polarity;

        public string activator;

        public float energy;

        /**
         * cvList for current scan
         */
        public List<CV> cvList;

        public double[] rts;
        public double[] ints;
    }
}

File: JavaSDK/src/main/java/net/csibio/aird/bean/common/MrmPair.java
package net.csibio.aird.bean.common;

import lombok.Data;
import net.csibio.aird.bean.CV;
import net.csibio.aird.bean.WindowRange;

import java.util.List;

@Data
public class MrmPair {

    /**
     * order number for current spectrum
     */
    int num;

    String id;

    String key;

    WindowRange precursor;
    WindowRange product;
    String polarity;
    String activator;
    Float energy;

    /**
     * cvList for current scan
     */
    List<CV> cvList;

    double[] rts;
    double[] ints;
}
